LoggerFinal/src/main/java/com/Logger/utils/CriteriaChecking.java

```diff
@@ -0,0 +1,57 @@
+package com.Logger.utils;
+
+import org.springframework.data.mongodb.core.query.Criteria;
+
+import java.util.List;
+
+/**
+ * Created by technology on 8/5/17.
+ */
+public class CriteriaChecking {
+
+    public static void nullCheck(List<Criteria> criteriaList, RoomAvailabilityRequest request) {
+        if (request.startDate == null && request.endDate == null && (request.hotelId == null || request.hotelId == 0)
+                && (request.roomId == null || request.roomId == 0) && request.reservationId == null && (request.rateplanId==null || request.rateplanId==0) ) {
```

Comment:
If an user try to give 0 then even that should be handled since roomId cannot be 0 ryt?